code/modules/client/client procs.dm

```diff
@@ -212,7 +212,7 @@
 		if("openLink")
 			src << link(href_list["link"])
 
-	..()	//redirect to hsrc.Topic()
+	..()	//redirect to hTopic()
```

Comment:
You need to use a better find-replace regex for this. *And* actually double-check the diff afterwards.